src/blk/spdk/NVMEDevice.cc

```diff
@@ -588,10 +588,6 @@ int NVMEManager::try_get(const spdk_nvme_transport_id& trid, SharedDriverData **
         spdk_env_init(&opts);
         spdk_unaffinitize_thread();
 
-        spdk_nvme_retry_count = g_ceph_context->_conf->bdev_nvme_retry_count;
```

Comment:
could you remove the `bdev_nvme_retry_count` option in this change?

Reply:
I removed it in current PR, first patch named "os/bluestore: remove nvme retry count configuration from NVMEDevice"

Reply:
@rouming i mean, to remove it from `src/common/legacy_config_opts.h` and `src/common/options.cc`. as you are ditching its only consumer.

Reply:
@tchaikov  Ah, ok, misunderstood. I did not remove the option in order to keep compatibility. Will remove if you confirm, that osd won't die if someone still provides the option.

Reply:
@rouming no, i don't think osd will crash with this change. moreover, SPDK backend is still experimental. so i think it's fine removing its options.